### PR TITLE
Adding 5 resources, 1 tag

### PR DESCRIPTION
**Pull request by andy5995**
*added "veterans' tag
*added resource: official VA Mental Health site
*added resource: US warmline directory

### CODE REVIEW

**Comment by andy5995** [comment]
Looks like the ci/circleci check failed the first time. I tried to fix it with the second commit, but the check timed out. I'd be happy to make any necessary changes, just let me know.

File: config/locales/ptbr.yml [hunk 1]
```diff
@@ -475,6 +475,7 @@ ptbr:
         mindfulness: paz de espírito
         journaling: Diário
         mood_tracking: Registro de alterações de humor
+	veterans: veteranos
```
Comment (timorthi) [inline config/locales/ptbr.yml]: Looks like the CI build is failing because this line is indented by tabs instead of spaces?

Comment (andy5995) [reply config/locales/ptbr.yml]: Thank you. Fixed.

**Comment by andy5995** [comment]
Thanks for reviewing, @timorthi . And sure, I'll wait for 585.

**Comment by julianguyen** [comment]
Gonna merge this first, since there are other translation keys from other merged PRs which needs Swedish translations too! @alexiamcdonald, please rebase :)

**Pull request merged**